src/main/java/net/bramp/ffmpeg/builder/AbstractFFmpegStreamBuilder.java

```diff
@@ -583,6 +583,10 @@ protected List<String> build(FFmpegBuilder parent, int pass) {
       args.add("-sn");
     }
 
+    if(parent.format != null){
+      addFormatArgs(args);
+    }
+
```

Comment:
@bramp 

I think how about adding code like this so we can add argument for format?

If there is an output builder for a new format in the future, we can override and add the arguments of each format

Reply:
nit: Call this addFormat**Flags** (like addAudioFlags, addVideoFlags, etc)

Something like that may make it easier, especially when the position of the arguments matter.

I'd want to put all format related flags in this section, such as "-f format", and then in HlsOutputBuilder you override `addFormatFlags` and call super.addFormatFlags.  Just quickly looking I think `-f format` is the only format related flag I see.

Reply:
Oh! Without the format, it wouldn't be a FormatOutputBuilder?? Thanks.